core/src/main/java/org/apache/carbondata/core/metadata/schema/table/CarbonTable.java

```diff
@@ -18,7 +18,13 @@
 package org.apache.carbondata.core.metadata.schema.table;
 
 import java.io.Serializable;
-import java.util.*;
+import java.util.ArrayList;
```

Comment:
Use import java.util.*  for better readability and also guidelines suggests that more than 4 import from the same package 
should include *

Reply:
ok